styles/styles.css

```diff
@@ -0,0 +1,337 @@
+}
+
+#letterhead {
+  display: flex;
+  flex-direction: column;
+  
+  margin-top: 3%;
+  color: #81B642;
+  font-family: Merriweather, Times, serif;
+  font-style: italic;
+  text-align: center;
+  font-size: 150%;
+}
+
+.filler {
+  width: 0%;
+  
+ 
+}
+
+#toplogo {
+  max-width: 100%;
+  height: auto;
+}
+
+#cart {
+  display: flex;
+  flex-direction: column;
+  justify-content: right;
+  margin-top: 2rem;
+   
+}
+
+.top-cart {
+  color: white;
+  font-family: "Archivo Black", Geneva, sans-serif;
+  font-weight: bold;
+}
+
+
+#carticon {
+/*Size to make small; I've just shrunk it a bit for now*/
+width: 1.5em;
+
+}
+
+.checkout-button {
+  /*Green background, position beneath cart icon */
+    background-color: #81B642;
+    height: 30px;
+    width: 140px;
+    color: white;
+    font-family: "Archivo Black", Geneva, sans-serif;
+    font-weight: bold;
+    font-size: 75%;
+    line-height: 30px;
+    text-align: center;
+    margin: 10px;
+}  
+
+h1 {
+  
+/*Size */
+}
+
+#subbanner {
+  background-color: #ebebeb;
+  height: 4em;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+.subtitle {
+ /*Position away from left margin and check font */
+  color: #093b60;
+  font-family: "Archivo Black", Geneva, sans-serif;
+  font-weight: bold;
+  margin-left: 10rem;
+}
+
+#welcome {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 1rem;
+}
+
+#hugpicture {
+  width: 85%;
+}
+
+h2 {
+  font-family: "Palatino Linotype", "Book Antiqua", Palatino, serif;
+  color: #093b60;
+  font-size: 1.75em;
+  font-weight: 900;
+}
+
+.lead-in {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  color: #3C3C3C;
+  width: 30rem;
+
+}
+
+.profiles {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+  justify-content: center;
+}
+
+.profile-card {
+  width: 200px;
+  height: 375px;
+  margin: 25px;
+  border: 2px solid #ebebeb;
+}
+
+.profilepicture {
+  width: 200px;
+}
+
+h3 {
+  font-family: "Palatino Linotype", "Book Antiqua", Palatino, serif;
+  color: #093b60;
+  font-size: 1.25em;
+  font-weight: 900;
+}
+
+.cost {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: .8em;
+  color: #3C3C3C;
+}
+
+.bio {
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  font-size: .8em;
+  color: #3C3C3C;
+}
+
+.adopt-button {
+  background-color: #81B642;
+  height: 30px;
+  width: 180px;
+  color: white;
+  font-family: "Archivo Black", Geneva, sans-serif;
+  font-weight: bold;
+  font-size: 90%;
+  line-height: 30px;
+  text-align: center;
+  margin: 10px;
+}
+
+a.checkout:visited {
+  color: white;
+}
+
+/*Not working: make adopt-button white and still white when visited*/
+
+.superfooter {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  align-items: center;
+  justify-content: center;
+  height: 10rem;
+}
+
+#whitespace {
```

Comment:
Get rid of this in the HTML. See if you can find another way to add that spacing. We don't want to add elements in the page to add blank space.